modules/imgcodecs/test/test_read_write.cpp

```diff
@@ -5,6 +5,151 @@
 
 namespace opencv_test { namespace {
 
+TEST(Imgcodecs_Image, imread_reduce_flags)
```

Comment:
Please add `#ifdef HAVE_JPEG` guard for these tests (to avoid test failures in some configurations).

Reply:
Sure @alalek, will do